### PR TITLE
added template code frontend

### PR DESCRIPTION
**Pull request by ashish-patwal**

### CODE REVIEW

File: application/frontend/template/template.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        </p>
+      </div>
+      <footer class="footer">
+        <div class="container">
+          <p>&copy; 2023 Code Crafters. Made with ❤️.</p>
+        </div>
+      </footer>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.3/dist/umd/popper.min.js"></script>
```
Comment (gururani-abhishek) [inline application/frontend/template/template.html:73]: do we require these two external libs, read what Pavan Sir has to say.

File: application/frontend/assets/images/watchdogs.png
Comment (gururani-abhishek) [inline application/frontend/assets/images/watchdogs.png:1]: what is the use of this img? is it required?

File: application/frontend/template/template.html (hunk 1 above)
Comment (ashish-patwal) [reply application/frontend/template/template.html:73]: yes for collapsable navbar

File: application/frontend/assets/images/watchdogs.png
Comment (ashish-patwal) [reply application/frontend/assets/images/watchdogs.png:1]: just a logo for our team

**Comment by ashish-patwal** [comment]
yo should we merge ?
@gururani-abhishek

**Comment by gururani-abhishek** [comment]
> yo should we merge ?
> @gururani-abhishek

Bro as discussed, we should close this PR.

**Pull request closed without merging**